src/style.css

```diff
@@ -1,8 +1,14 @@
-body {
-  font-family: Arial;
+html {
+  font-family: Arial, sans-serif;
   font-size: 13px;
 }
 
+h1,
+h2,
+p {
+  margin: 0;
+}
+
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice